homeassistant/components/fronius/__init__.py

```diff
@@ -195,8 +247,19 @@ async def _get_inverter_infos(self) -> list[FroniusDeviceInfo]:
                     unique_id=unique_id,
                 )
             )
+            _LOGGER.debug(
+                "Inverter found at %s (Device ID: %s, UID: %s)",
+                self.host,
+                solar_net_id,
+                unique_id,
+            )
         return inverter_infos
 
+    @callback
+    async def _rescan_inverter(self, now: datetime) -> None:
+        """Initiate a scheduled rescan of available inverters."""
+        await self._init_devices_inverter()
+
```

Comment:
You don't need the decorator if it is async anyway. 

And since this method now just wraps an await, you can probably omit it completely and use `self._init_devices_inverters` in `async_track_time_interval`.

Reply:
Yes, hat'll work fine and save some lines of code.